lesson04/assignment/script/script.js

```diff
@@ -0,0 +1,190 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // create array for different pieces of Art
+  // temporarily using my own pieces, till time to dig into API
+  const art = [
+      {
```

Comment:
Nice work importing all of these as JS object. (I like the Dia de los Muertos / Nightmare before Christmas combo)